### ISSUE TITLE
C# grpc 1.2.0: VCRUNTIME140.dll is missing

### ISSUE THREAD

**Issue by vadimi** [grpc/grpc#10251]
I'm running grpc 1.2.0 service on Windows Server 2012R2 and it throws the following exception:

![image](https://cloud.githubusercontent.com/assets/1156106/24206459/af3d6dd2-0ef4-11e7-9144-8c754965aa50.png)

The same service runs just fine using grpc 1.1.0.

And that's the exception it throws:

```
Unhandled Exception: System.IO.IOException: Error loading native library "C:\Temp\grpc\net461\grpc_csharp_ext.x86.dll"
```

**Comment by vadimi** [comment]
So it looks like it doesn't statically link MSVC++ runtime after this pull request: https://github.com/grpc/grpc/pull/9568

**Comment by jtattermusch** [comment]
Thanks for filing this. The workaround is to install https://www.microsoft.com/en-us/download/details.aspx?id=48145 (can you please confirm that fixes it?).

Actually, I am not 100% sure that the runtime wasn't required even before, but we switched the build from VS2013 (which would require the VS2013 C++ redistributable) to VS2015 - hence the difference.

Btw, our cmake file already supports gRPC_MSVC_STATIC_RUNTIME option, so just enabling it for grpc_csharp_ext build can fix this (but I am not 100% sure if statically linked runtime is what we actually want):
https://github.com/grpc/grpc/blob/6f1e443a519cd28d97be78c5ca2ca72a45f6b598/cmake/msvc_static_runtime.cmake#L1

**Comment by jtattermusch** [comment]
@vadimi    #9568 doesn't seem to be what actually disables static linking, it just adds the option to enable it. The actual change seems to be  https://github.com/grpc/grpc/pull/9665.

**Comment by jtattermusch** [comment]
So before #9665, we were using `<RuntimeLibrary>MultiThreaded</RuntimeLibrary>` here:
https://github.com/grpc/grpc/blob/6f1e443a519cd28d97be78c5ca2ca72a45f6b598/vsprojects/vcxproj/grpc_csharp_ext/grpc_csharp_ext.vcxproj#L123

According to https://msdn.microsoft.com/en-us/library/aa278396(v=vs.60).aspx that seems to translate to:
`/MT` and `/MTd` options, which is what gRPC_MSVC_STATIC_RUNTIME enables?

**Comment by vadimi** [comment]
Thanks for explanation, I'll check out the pull request.

**Comment by jtattermusch** [comment]
This has been fixed by #10270.

Just pushed nugets with the fix (version 1.2.2).

**Comment by vadimi** [comment]
thanks a lot!
